compiler/pom.xml

```diff
@@ -32,6 +32,16 @@
       <artifactId>auto-value</artifactId>
       <optional>true</optional>
     </dependency>
+    <dependency>
+      <groupId>org.jgrapht</groupId>
+      <artifactId>jgrapht-core</artifactId>
+      <version>0.9.1</version>
+    </dependency>
+    <dependency>
+      <groupId>com.google.guava</groupId>
```

Comment:
`com.google.guava:guava` needs to be excluded from shading (see maven-shade-plugin configuration below)